lnd.go

```diff
@@ -30,6 +30,7 @@ import (
 
 	"golang.org/x/net/context"
 
+	"golang.org/x/crypto/acme/autocert"
```

Comment:
nit: could be grouped with above import. tbh I don't know if there is any structure behind the imports in this file 😛

Reply:
We definitely need an import robot. Or alternatively don't care about import order in reviews (my pref) and just touch up whenever you feel like it.

Made three groups now in this file: base libs, other libs and lnd libs.

Reply:
See https://github.com/lightninglabs/faraday/pull/74, maybe this is something we'd want to use in `lnd` too?

Reply:
Possibly, but not in this PR :)

Reply:
No, of course not in this PR. But if you agree with the proposed idea, you could order the imports that way already, just an idea.

Reply:
Yes, I agree with the idea. Added commit.